build/build-image.sh

```diff
@@ -21,51 +21,25 @@ ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")"/.. >/dev/null && pwd)"
 
 CORTEX_VERSION=master
 
-slim="false"
-while [[ $# -gt 0 ]]; do
-  key="$1"
-  case $key in
-    --include-slim)
-    slim="true"
-    shift
-    ;;
-    *)
-    positional_args+=("$1")
-    shift
-    ;;
-  esac
-done
-set -- "${positional_args[@]}"
-
-dir=$1
-image=$2
+image=$1
+dir="${ROOT}/images/${image/-slim}"
 
 if [ "$image" == "inferentia" ]; then
   aws ecr get-login-password --region us-west-2 | docker login --username AWS --password-stdin 790709498068.dkr.ecr.us-west-2.amazonaws.com
 fi
 
-docker build "$ROOT" \
-  -f $dir/Dockerfile \
-  -t cortexlabs/${image} \
-  -t cortexlabs/${image}:${CORTEX_VERSION}
-
-if [ "$slim" == "true" ]; then
-  if [ "${image}" == "python-predictor-gpu" ]; then
-    cuda=("10.0" "10.1" "10.2" "11.0")
-    cudnn=("7" "7" "7" "8")
+build_args=""
+if [[ "$image" == *"-slim"* ]]; then
```

Comment:
nit: does it work if the expression is evaluated from `"$image" == *"-slim"`?

Reply:
I added the `*` at the end since I thought that maybe in the future we could have more suffixes which could come after. But we could also make the change then if that happens, what do you think?

Reply:
I see. For strictness purposes, I'd lean towards not having that `*` at the end. I don't exactly have a strong opinion on this, so whichever you think makes more sense.